dask/bytes/tests/test_local.py

```diff
@@ -70,6 +70,17 @@ def test_with_urls():
         assert sum(map(len, values)) == len(files)
 
 
+def test_with_paths():
+    pathlib = pytest.importorskip('pathlib')
+    with filetexts(files, mode='b'):
+        url = pathlib.Path('./.test.accounts.*')
+        sample, values = read_bytes(url, blocksize=None)
+        assert sum(map(len, values)) == len(files)
+    with pytest.raises(OSError):
```

Comment:
Normally (in this test) it's `IOError("%s resolved to no files" % urlpath)` in read_bytes, but it could be some form of "path not accessible".